internal/manifest/parse.go

```diff
@@ -115,3 +146,60 @@ func pullLayer(ctx context.Context, imageRef string, keyChain authn.Keychain) (c
 func getFsChartPath(imageSpec v1beta2.ImageSpec) string {
 	return filepath.Join(os.TempDir(), fmt.Sprintf("%s-%s", imageSpec.Name, imageSpec.Ref))
 }
+
+func untarLayer(tarPath string) (string, error) {
+	tarFile, err := os.Open(tarPath)
+	if err != nil {
+		return "", fmt.Errorf("failed to open file: %w", err)
+	}
+	defer tarFile.Close()
+
+	tarReader := tar.NewReader(tarFile)
+
+	for {
+		header, err := tarReader.Next()
+		if errors.Is(err, io.EOF) {
+			break
+		}
+		if err != nil {
+			return "", fmt.Errorf("failed to read tar: %w", err)
+		}
+
+		if strings.HasPrefix(header.Name, "._") {
```

Comment:
what's the purpose of this check?

Reply:
But then it means the content in this tar file already invalid, we don't need to tolerate this misconfigured tar file, please don't introduce such hacking, instead, we should prevent it while generating this layer.

Reply:
I left a comment regarding this, it should only affect in Mac OS